l5kit/l5kit/geometry/transform.py

```diff
@@ -71,46 +71,70 @@ def flip_y_axis(tm: np.ndarray, y_dim_size: int) -> np.ndarray:
 
 
 def transform_points(points: np.ndarray, transf_matrix: np.ndarray) -> np.ndarray:
-    """Transform points using transformation matrix.
-    Note this function assumes points.shape[1] == matrix.shape[1] - 1, which means that the last
-    row in the matrix does not influence the final result.
-    For 2D points only the first 2x3 part of the matrix will be used.
+    """
+    Transform points using transformation matrices with 3 modes:
+    - points (N, f), transf_matrix (f+1, f+1)
+        all points are transformed using the matrix and output has shape (N,f).
```

Comment:
```suggestion
        all points are transformed using the matrix and the output points have shape (N, f).
```

Reply:
@lucabergamini does `f` stands for `feature`? Should we use  capitalized form `F`?

Reply:
Ping, should we use `F` for the last feature dimension?

Reply:
@lucabergamini